essential.md

```diff
@@ -161,13 +167,9 @@ Image description is encoded as
 
 - The top-level `rdf:Description`
 - shall contain 0 or 1 `dc:description`
-    - which shall contain 1 `rdf:Alt`
-    - which shall contain 1 or more `rdf:li`, each with the `xml:lang` attribute set to a distinct language tag
-    - each of which shall contain a free-text content in the given human language
-- and shall contain 0 or 1 `dc:title`
-    - which shall contain 1 `rdf:Alt`
-    - which shall contain 1 or more `rdf:li`, each with the `xml:lang` attribute set to a distinct language tag
-    - each of which shall contain a human-readable name for identifying this image
+- which shall contain 1 `rdf:Alt`
+- which shall contain 1 or more `rdf:li`, each with the `xml:lang` attribute set to a distinct language tag
+- each of which shall contain a free-text content in the given human language
 
 ## 2.3. Date   <a name="2.3"></a>
```

Comment:
I expected to see something about syncing with the IPTC and exif dates.

Also for locations, we need to spell out the synchronization rules to the legacy XMP location fields.